fan2go.yaml

```diff
@@ -146,10 +167,11 @@ curves:
       sensor: cpu_package
       # Steps to define a section-wise defined speed curve function
       steps:
-        # Sensor value (degrees Celsius) -> Speed (0-255)
-        - 40: 0
-        - 50: 50
-        - 80: 255
+        # Sensor value (degrees Celsius) -> Speed (0%: Fan stopped; 1%: MinPwm; 100%: MaxPwm)
+        - 40: 0%   # up to 40°C stop the fan (unless neverStop is true)
+        - 41: 1%   # at 41°C the fan should run at its minimum (non-stopped) speed
+        - 65: 33%  # at 50°C the fan should run at 33% speed
```

Comment:
there is a mismatch between the value (65) and the comment, should probably be 50?

Reply:
true